api/lib/opentelemetry/context/propagation/rack_env_getter.rb

```diff
@@ -32,15 +32,15 @@ def keys(carrier)
         private
 
         def to_rack_key(key)
-          ret = 'HTTP_' + key
+          ret = "HTTP_" + key # rubocop:disable Style/StringConcatenation, Style/StringLiterals
```

Comment:
Would you be able to provide a little more insight here?

What is the motivation for switching to double quotes?

Reply:
I think this was an oversight - apologies.

Reply:
Yup, oversight. Changed back.